exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `message`
-
+let message = "This is a string";
+let messageType = typeof message;
```

Comment:
you should use `const` instead of `let`, as you are not modifying the value of the variables after initizialisation.